amundsen_application/static/js/components/NavBar/index.tsx

```diff
@@ -49,7 +52,13 @@ export class NavBar extends React.Component<NavBarProps> {
                 <span className="title-3">AMUNDSEN</span>
               </Link>
             </div>
-            <div id="nav-bar-right" className="nav-bar-right">
+            {
+              this.props.location.pathname !== "/" &&
```

Comment:
This is probably fine for now, but in the future we might have more complex rules for when to show/hide the search bar.

Reply:
Will make a `renderSearchBar()` method.